schemas/action-schema.json

```diff
@@ -67,6 +67,20 @@
                 }
               }
             },
+            "crash": {
+              "type": "object",
+              "description": "Properties of the crashes of the action",
+              "required": [
+                "count"
+              ],
+              "properties": {
+                "count": {
+                  "type": "integer",
+                  "description": "Number of crashes that occurred on the action",
```

Comment:
Can we have multiple crashes on a single view?
I would assume that a single crash kills the application so reopen the application would start a new view.
What is the real behavior?

Reply:
Yes you're correct, once a view/action has a crash, it can't have more. 
IIRC, the discussion with Nicolas was that we want to mimic the behavior of error_count, and have a numeric instead of boolean value to ease integrating with dashboards.

Reply:
ok, I get it, thx!